docs/library/machine.RTC.rst

```diff
@@ -75,6 +75,15 @@ Methods
       - ``wake`` specifies the sleep mode from where this interrupt can wake
         up the system.
 
+.. method:: RTC.memory([bytes])
+
+   **Note**: ESP8266 and ESP32 only
```

Comment:
For consistency add as note at end "Availability: ESP32, ESP8266." eg. see [`machine.rst`](https://docs.micropython.org/en/latest/library/machine.html#machine.wake_reason).

Reply:
Thanks for review. done.